src/pluginlaunch.c

```diff
@@ -411,7 +412,11 @@ get_available_memory ()
     }
   len = fread (buf, 1, sizeof (buf) - 1, fd);
```

Comment:
Hi there, @nichtsfrei 
IMHO - Rather than just reading the file, I think it would be better to get the file size from the offset and then make sure that the fread() reads and check as normally as the file size 😆 

How about you?

Reply:
It may not be too important, but `char buf` is using around 8192 of arbitrary unnecessary memory.
so I think it's good to get the file size and then read it that much.